build_tools/circle/install.sh

```diff
@@ -30,9 +30,11 @@ popd
 # Configure the conda environment and put it in the path using the
 # provided versions
 conda create -n testenv --yes python pip pytest nose \
-   numpy scipy scikit-learn matplotlib
+   numpy scipy scikit-learn matplotlib cython
 source activate testenv
 
+pip install -r requirements.txt
```

Comment:
do we need this? Especially after #383 where we switched to using `pip install -e.` I think we should be good without this. (same in travis)